tlsutil/config.go

```diff
@@ -561,6 +567,16 @@ func (c *Configurator) VerifyServerHostname() bool {
 	return c.base.VerifyServerHostname || c.autoEncrypt.verifyServerHostname
 }
 
+// IncomingGRPCConfig generates a *tls.Config for incoming GRPC connections.
+func (c *Configurator) IncomingGRPCConfig() *tls.Config {
+	c.log("IncomingGRPCConfig")
+	config := c.commonTLSConfig(false)
```

Comment:
`false` has the effect that this config doesn't require client cert check. This is because we are missing a `verify_incoming_grpc` configuration option. And using `verify_incoming` would be backwards incompatible, because even if it was set before, it didn't have an effect on the grpc server.

Reply:
helpful context! wonder if it would be worth a comment in the code so future developer knows why not to change this. maybe it is apparent though.

Reply:
Good point, done!